docs/ref/models/fields.txt

```diff
@@ -417,8 +417,7 @@ guaranteed to fit numbers from ``-9223372036854775808`` to
 .. class:: BinaryField(max_length=None, **options)
 
 A field to store raw binary data. It only supports ``bytes`` assignment. Be
-aware that this field has limited functionality. For example, it is not possible
-to filter a queryset on a ``BinaryField`` value.
+aware that this field has limited functionality.
```

Comment:
If an example still exists, I'd like to add one. Although I'm not familiar if there is one and what it would be. Perhaps I should also remove "Be aware that this field has limited functionality.", WDYT?